src/apps/api/views/competitions.py

```diff
@@ -21,6 +21,7 @@
 
 class CompetitionViewSet(ModelViewSet):
     queryset = Competition.objects.all()
+    permission_classes = (AllowAny,)
```

Comment:
does this let anyone delete a competition, as well?

Reply:
✅